apps/storybook/postcss.config.js

```diff
@@ -1 +1,6 @@
-module.exports = require("@incmix/tailwind-config/postcss")
```

Comment:
This import gave an error for me. Of course this way would be better than copy pasting the config I did now.